portfolio/src/main/java/com/google/sps/data/User.java

```diff
@@ -30,4 +30,14 @@ public static String getUserNickname(String id) {
     }
     return (String) entity.getProperty("nickname");
   }
+  public static Integer getVote(String id) {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Vote").setFilter(new Query.FilterPredicate("id", Query.FilterOperator.EQUAL, id));
+    PreparedQuery results = datastore.prepare(query);
+    Entity entity = results.asSingleEntity();
+    if (entity == null) {
+      return null;
+    }
+    return (int) (long) entity.getProperty("slice");
```

Comment:
Why are you converting to first to long and then to int? 
Maybe you can also use Integer.parseInt to catch conversion errors?